pilot/pkg/networking/core/v1alpha3/route/route.go

```diff
@@ -428,6 +428,7 @@ func translateRoute(push *model.PushContext, node *model.Proxy, in *networking.H
 				action.RequestMirrorPolicy = &route.RouteAction_RequestMirrorPolicy{
 					Cluster:         GetDestinationCluster(in.Mirror, serviceRegistry[host.Name(in.Mirror.Host)], port),
 					RuntimeFraction: mp,
+					TraceSampled:    &wrappers.BoolValue{Value: false},
```

Comment:
This will break 1.5 proxies 

@linsun @ostromart here is a our first case where we need to decide if we support in place upgrade or not

Reply:
Actually I guess not since its an unknown field, and we are not relying on the behavior - so 1.5 proxy will gracefully ignore it

Reply:
The proxy here has not been updated yet, so we're actually testing the unknown field behavior in this PR.